### ISSUE TITLE
"Facility address audit" tainted multiple facility views.

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#10281]
## Describe the defect
IT looks like a "apply to all views" option was chosen during an update.   

![image](https://user-images.githubusercontent.com/5752113/184721710-1d14fdd4-aae5-4d79-9173-30abc5784d32.png)

The following views are showing the wrong title.
- [ ] /admin/content/facilities
- [ ] /admin/content/facilities/facility-status
- [ ] /admin/content/facilities/vamc-facility-health-services
- [ ] /admin/content/facilities/vamc-facility-non-clinical-services
- [ ] /admin/content/facilities/vamc-prepare-for-visit
- [ ] /admin/content/facilities/vamc-systems
- [ ] /admin/content/facilities/vamc-system-service-audit
- [ ] /admin/content/facilities/vet-center-services
## To Reproduce
1. Go to Any of the views listed
2. See that they all have the same subtitle of "Facility address audit"

## AC / Expected behavior
- [ ] All facility views listed above have their own titles
- [ ] This is the only view /admin/content/facilities/addresses that has this title   "Facility address audit"

## Additional context
Add any other context about the problem here. Reach out to the Product Managers to determine if it should be escalated as critical (prevents users from accomplishing their work with no known workaround and needs to be addressed within 2 business days).

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by omahane** [comment]
@swirtSJW 
It turns out that a title was not set for all the Facilities view displays. It is instead the table caption, which will be the same on every display. I recommend we use something generic to all the displays of this view: "Facility query results"
<img width="1307" alt="Screen Shot 2022-09-01 at 9 16 01 AM" src="https://user-images.githubusercontent.com/766573/187936969-bbbdf4fd-1994-49e1-8239-ee6918d00f46.png">

**Comment by swirtSJW** [comment]
I looked for an issue on D.o related  to this and was surprised not to find one.  I raised it in Civic Actions slack to Mike G who was instrumental in getting that functionality to core to just double check me on this being a bug.  (seems like a core Views bug to me)

A single caption that is only right one time out of 1/9th of the time seems like a bug to me. 
![image](https://user-images.githubusercontent.com/5752113/190550865-0265a25f-9376-48b1-b7fc-da8e086028f1.png)

I like that in your PR you have made it more generic so it is right more of the time, but I question the value in having it, if it is just repeated, half helpful noise that is misleading in its ambiguity.  I wonder if just having it blank is better than having it present.. @laflannery  may be able to provide some additional guidance.

**Comment by laflannery** [comment]
All data tables must have a caption, without this screen readers have no way of understanding and announcing the purpose of the table. That being said, a generic caption for all tables doesn't fully explain the data within the table but I do feel it is the best option for now until the core bug is solved.

**Comment by laflannery** [comment]
After speaking with @swirtSJW and @omahane we decided to remove the incorrect caption rather than add the generic caption.  While it is a best practice to have the caption however, not having it is not actually 508 or WCAG violation. These views are also not editor facing and our current prioritization for accessibility audit/remediation is on the editor experience. We feel that the best use of effort would be to remove the incorrect captions and post this as an issue on d.o.

**Comment by omahane** [comment]
It turns out that overriding the Format section of the view display also overrides the Settings of the Format, in which the caption is set. Therefore, we can override the caption for each display in the same view. For the Facilities view, I propose the following captions as a bullet under their respective displays.

_Note: I used the "list of" nomenclature to identify a group of related items._
Facilities
- List of facilities in the CMS
Addresses
- List of facilities in the CMS
Facility Status(active tab)
- List of facilities and their status
VAMC facility health services
- List of health services at  VAMC facilities
VAMC facility non-clinical services
- List of non-clinical services at VAMC facilities
VAMC facility "Prepare for your visit"
- List of accordion items in the "Prepare for your visit" field of the VAMC facility content type
VAMC systems
- List of VAMC Health Systems in the CMS
VAMC System Service
- List of health services at VACM health systems
Vet Center services
- List of services at Vet Centers

**Comment by swirtSJW** [comment]
"List of health services at VACM health systems"
could probably be
"List of health services at VAMC systems"

**Comment by swirtSJW** [comment]
"List of accordion items in the "Prepare for your visit" field of the VAMC facility content type"
 could become
 "List of accordion items in the "Prepare for your visit" field on VAMC facilities"

**Comment by laflannery** [comment]
These look good to me!
